app/mailers/remove_user_mailer.rb

```diff
@@ -0,0 +1,17 @@
+class RemoveUserMailer < ApplicationMailer
+  # RemoveUserMailer.send_email.deliver_later
+  def send_email(user, organisation)
+    mailer_options = {
+      to: user.email,
+      subject: t(
+        "#{user.service}.mailers.remove_user.subject",
+        organisation_name: organisation.name,
+      ),
+      body: t("#{user.service}.mailers.remove_user.body",
+              user_name: user.full_name,
+              organisation_name: organisation.name),
+    }
+
+    notify_email(mailer_options)
+  end
+end
```

Comment:
I'm wondering if this mailer, and the [existing "user invite" mailer](https://github.com/DFE-Digital/itt-mentor-services/blob/bf8bde00122b6b4d52552d97f8e71423e667e51b/app/mailers/notify_mailer.rb#L3-L17), would be better off grouped into a single mailer.

For example:

```ruby
class UserMailer < ApplicationMailer
  def invitation_email(user, organisation)
    # do something
  end

  def removal_email(user, organisation)
    # do something
  end
end
```

This seems like a reasonable grouping to make, and follows the Rails guidance which says that "[mailers are similar to controllers](https://guides.rubyonrails.org/action_mailer_basics.html#mailers-are-similar-to-controllers)" in that they can have multiple "actions".

What do you think?

Reply:
This seems sensible to me! Will make the change / refactor the other mailer